meta/saisanitycheck.c

```diff
@@ -1661,6 +1666,11 @@ void check_attr_acl_fields(
                 break;
             }
 
+            if (md->objecttype == SAI_OBJECT_TYPE_DTEL)
+            {
+                break;
+            }
+
```

Comment:
why is this needed?

Reply:
Because I used sai_acl_field_data_t for [SAI_DTEL_ATTR_INT_L4_DSCP](https://github.com/xiaozhou/SAI/blob/master/inc/saidtel.h#L143-L151) as Kamil suggested. It is same as the special case UDF MATCH in lines above. Should I make it more detailed with specific fields allowed which is similar to the UFD MATCH?

Reply:
please remove this check, there is something wrong here it captures too much data scope,
whats the error when you remove this ?

Reply:
if you want to make it simlar to udf match, please use only explicit attributes and specific valu types like for udf match in lines 1637..1662

Reply:
The error messages would be: *ASSERT FAILED (on line 1669): SAI_DTEL_ATTR_INT_L4_DSCP: acl field may only be set on acl field and udf match*.

I have updated the check to only match on specified attr and value types